subt/docker/robotika/Dockerfile

```diff
@@ -9,11 +9,31 @@ RUN /usr/bin/python3 -m pip install --user --no-cache-dir \
   "pyzmq==19.0.0"
 
 RUN cd /home/developer/ && head -n -2 .bashrc > .bashrc.new && mv .bashrc.new .bashrc
+RUN sudo apt update
 
 COPY --chown=developer:developer . ./osgar/
 
 RUN source /opt/ros/melodic/setup.bash && ./osgar/subt/script/sync.sh
 
+#COPY subt_seed_node.cc ./src/subt_seed/src/subt_seed_node.cc
+#COPY CMakeLists.txt ./src/subt_seed/CMakeLists.txt
+
+ADD ./subt/docker/robotika/ros ./src
+
+RUN sudo apt install -y ros-melodic-image-geometry \
+			ros-melodic-pcl-ros \
+			ros-melodic-costmap-2d \
+			libdxflib-dev \
+			ros-melodic-rtabmap-ros \
+			ros-melodic-teleop-twist-keyboard \
+			ros-melodic-move-base \
+			ros-melodic-dwa-local-planner \
+			ros-melodic-robot-localization \
+			ros-melodic-depthimage-to-laserscan \
+			mc
+
+RUN /bin/bash -c 'source /opt/ros/melodic/setup.bash && catkin_make install'
```

Comment:
`catkin_make` is called inside `sync.sh` already. Also we no longer call `install`. I think we need to look again on the order of commands in this `Dockerfile`.

Reply:
ok